traitsui/wx/custom_editor.py

```diff
@@ -57,10 +54,7 @@ def init(self, parent):
         """
         factory = self.factory.factory
         if factory is not None:
-            try:
-                self.control = factory(*((parent, self) + self.factory.args))
-            except:
-                open_fbi()
+            self.control = factory(*((parent, self) + self.factory.args))
```

Comment:
Arguably the equivalent here should be to do a `try: ... except: pass`, or at least a `try: ... except Exception: pass`, but the corresponding Qt code doesn't try to intercept exceptions here.

Reply:
It is a behaviour change, but I think it is OK.